e2e/README.md

```diff
@@ -0,0 +1,25 @@
+This directory contains a suite of black box and ui integration tests.
```

Comment:
I would add steps on how to run locally to the README. Feel free to leave the relevant TODOs

I'm currently running tests with `yarn start` and `yarn cypress run` (in another shell).  The quickstart tests are failing for me right now. Need to look into it more.